File: feraltweaks/API/Networking/IModNetworkHandler.cs
namespace FeralTweaks.Networking
{
    /// <summary>
    /// Interface used by FeralTweaks to bind mods to the networking system of Fer.al
    /// </summary>
    public interface IModNetworkHandler
    {
        /// <summary>
        /// Retrieves the network messenger used to handle and send packets for this mod
        /// </summary>
        /// <returns>ClientMessenger instance</returns>
        public ClientMessenger GetMessenger();
    }
}

File: feraltweaks-server-module/src/main/java/org/asf/centuria/feraltweaks/api/networking/IModNetworkHandler.java
package org.asf.centuria.feraltweaks.api.networking;

/**
 * 
 * Interface used by FeralTweaks to bind mods to the server networking system
 * 
 * @author Sky Swimmer
 *
 */
public interface IModNetworkHandler {

	/**
	 * Retrieves the network messenger used to handle and send packets for this mod
	 * 
	 * @return ServerMessenger instance
	 */
	public ServerMessenger getMessenger();

}
